examples/GPIO/Blink/Blink.ino

```diff
@@ -0,0 +1,40 @@
+/*  Example of reading and writing USB Host Shield GPI output
+    This example uses "Wiring" style interface. See Blink_LowLevel for example of using low-level UHS interface
+    Author: Brian Walton (brian@riban.co.uk)
+*/
+#include <UHS2_gpio.h>
+
+// Satisfy the IDE, which needs to see the include statment in the ino too.
+#ifdef dobogusinclude
+#include <spi4teensy3.h>
+#endif
+
+#define OUTPUT_PIN 0
+
+USB Usb; //Create an UHS2 interface object
+UHS2_GPIO GPIO(&Usb); //Create a GPIO object
+
+void setup()
+{
+  Serial.begin( 115200 );
+#if !defined(__MIPSEL__)
+  while (!Serial); // Wait for serial port to connect - used on Leonardo, Teensy and other boards with built-in USB CDC serial connection
+#endif
+  Serial.println("Start");
+
+  if (Usb.Init() == -1)
+    Serial.println("OSC did not start.");
+
+  delay( 200 );
+}
+
+void loop()
+{
+  //Get the current output value, toggle then wait half a second
+  int nValue = GPIO.digitalReadOutput(OUTPUT_PIN);
+  nValue = (nValue?0:1);
```

Comment:
Maybe we should replace this with:

```C++
nValue = nValue ? LOW : HIGH;
```

So it is a little easier for beginners.